CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Help and issues
+
+Found a problem or need some help? The following steps may help:
+
+1. Make sure you have gone through the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki). A lot of questions have an answer in there!
+2. Upgrade to the latest version of the plugin and a recent Spigot/Bukkit build for your Minecraft version.
+3. Check your server's logs, there may be warnings/errors/stack straces to help you out with your issue!
```

Comment:
I would mention that it would be useful for developers if the stack trace was contained with code tags in the issue.

Reply:
Well steps 1 to 4 are supposed to help out _before_ actually opening the issue. In practice issues quite rarely involve stack traces with this project, so I'll leave that out. ^^